### PR TITLE
os/driver/lwnl: fix lwnl build error

### PR DESCRIPTION
**Pull request by an4967**
- fix build error in BLE lwnl

### CODE REVIEW

**Review by jeongchanKim — APPROVED** [review]
LGTM

File: framework/src/ble_manager/ble_manager_lwnl_listener.c [hunk 1]
```diff
@@ -97,10 +97,10 @@ int lwnl_fetch_ble_event(int fd, void *buf, int buflen)
...
 		/* Remove wifi data in socket. */
-		char *buf = (char *)malloc(len);
-		int nbytes = read(fd, (char *)buf, len);
-		BLE_LOG_VERBOSE("nbytes : %d / len : %d\n", nbytes, len);
-		free(buf);
+		char *t_buf = (char *)malloc(len);
+		nbytes = read(fd, (char *)t_buf, len);
+		BLE_LOG_INFO("read unused data nbytes : %d / len : %d\n", nbytes, len);
+		free(t_buf);
 		return;
```
Comment (jeongchanKim) [inline framework/src/ble_manager/ble_manager_lwnl_listener.c:104]: Please check the return type. The return type of this function is `int`, so it is better to return integer value.

File: framework/src/wifi_manager/wifi_manager_lwnl_listener.c [hunk 2]
```diff
@@ -165,10 +165,10 @@ int lwnl_fetch_event(int fd, void *buf, int buflen)
...
 		(void)_lwnl_call_event(fd, status, len);
 	} else {
-		char *buf = (char *)malloc(len);
-		int res = read(fd, buf, len);
-		NET_LOGI(TAG, "read unused data in the socket[%d]\n", res);
-		free(buf);
+		char *t_buf = (char *)malloc(len);
+		nbytes = read(fd, t_buf, len);
+		NET_LOGI(TAG, "read unused data in the socket[%d]\n", nbytes);
+		free(t_buf);
```
Comment (jeongchanKim) [inline framework/src/wifi_manager/wifi_manager_lwnl_listener.c:171]: In `ble_manager_lwnl_listener.c`, there is return after `free`. But in here, there is no.
Is it okay?

Comment (an4967) [reply framework/src/wifi_manager/wifi_manager_lwnl_listener.c:172]: I fixed it

File: framework/src/ble_manager/ble_manager_lwnl_listener.c (hunk 1 above)
Comment (an4967) [reply framework/src/ble_manager/ble_manager_lwnl_listener.c]: I fixed it

**Review by pillip8282 — APPROVED** [review]
lgtm

**Pull request merged**